package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "browserstack-tu-webdriverio",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
need to have one more default spec triggered if user only inputs "npm test", the default test should be the e2e.spec.js test